### PR TITLE
2912 paris cleanup

### LINKED ISSUE #2912: [Paris] clean up paris branch
clean up paris branch before release

- [x]  update all test only run in oxfordnet to run in paris and alpha to (chaning oxford/oxfordAndAlpha to it/test)
- [x]  replace where reference oxford to reference paris in all places

will leave test-dapp paris update to Ali where he'll add staking test cases along with paris changes
~~- [ ]  update test-dapp with paris reference and deploy contract on parisnet~~

### PR DESCRIPTION
**Pull request by hui-an-yang**
closes #2912 

Thank you for your contribution to Taquito.

Before submitting this PR, please make sure:

- [x] Your code builds cleanly without any errors or warnings
- [x] You have run the linter against the changes
- [ ] You have added unit tests (if relevant/appropriate)
- [ ] You have added integration tests (if relevant/appropriate)
- [ ] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
- [x] You have added or updated corresponding documentation
- [ ] If relevant, you have written a first draft summary describing the change for inclusion in Release Notes. 

In this PR, please also make sure: 

- [x] You have linked this PR to the issue by putting `closes #TICKETNUMBER` in the description box (when applicable)
- [ ] You have added a concise description on your changes
## Release Note Draft Snippet

__If relevant, please write a summary of your change that will be suitable for
inclusion in the Release Notes for the next Taquito release.__

### CODE REVIEW

File: integration-tests/__tests__/contract/operations/failing-noop.spec.ts [hunk 1]
```diff
@@ -43,8 +39,8 @@ CONFIGS().forEach(({ rpc }) => {
     let Mainnet: TezosToolkit;
 
     beforeAll(async () => {
-      Mainnet = new TezosToolkit('https://rpc.tzkt.io/mainnet'); // this is a mainnet archive history mode public rpc node url
-      Mainnet.setSignerProvider(new InMemorySigner(aliceSKey));
+      Mainnet = new TezosToolkit('https://rpc.tzkt.io/mainnet'); // public archive node to fetch genesis block
```
Comment (dsawali) [inline integration-tests/__tests__/contract/operations/failing-noop.spec.ts:42]: a bit misleading, change to 'mainnet public archive node to fetch older blocks'

File: integration-tests/__tests__/wallet/failing-noop.spec.ts [hunk 2]
```diff
@@ -3,21 +3,17 @@ import { OpKind, TezosToolkit } from "@taquito/taquito";
...
     beforeAll(async () => {
-      if(rpc.includes('oxfordnet')){
-        Tezos = new TezosToolkit('https://rpc.tzkt.io/oxfordnet');
-      } else if(rpc.includes('ghostnet')){
-        Tezos = new TezosToolkit('https://rpc.tzkt.io/ghostnet');
-      } else {
-        Tezos = new TezosToolkit(rpc);
+      setup(true)
+      if (rpc.includes('parisnet')) {
+        Tezos.setProvider({ rpc: 'https://rpc.tzkt.io/parisnet' }); // public archive node to fetch genesis block
```
Comment (dsawali) [inline integration-tests/__tests__/wallet/failing-noop.spec.ts:13]: same here and all other instances of the comment

**Review by dsawali — CHANGES_REQUESTED** [review]

**Comment by dsawali** [comment]
do you mind also updating the `parisAndAlpha()` methods to `it()` blocks in `staking.spec.ts` integration test?

File: integration-tests/__tests__/contract/operations/failing-noop.spec.ts (hunk 1 above)
Comment (hui-an-yang) [reply integration-tests/__tests__/contract/operations/failing-noop.spec.ts:42]: discussed on slack will not update to `fetch older blocks` as I find `fetch genesis block` more specific

**Comment by hui-an-yang** [comment]
> do you mind also updating the `parisAndAlpha()` methods to `it()` blocks in `staking.spec.ts` integration test?
@dsawali ghostnet is still in oxford protocol, will update them after transitioned. 
Will merge this pr shortly

**Pull request merged**